modules/041-linstor/monitoring/prometheus-rules/drbd-version.yaml

```diff
@@ -0,0 +1,22 @@
+- name: kubernetes.drbd.device_state
+  rules:
+    - alert: D8DRBDVersionIsOutdated
+      expr: drbd_version{kmod!="9.2.3"}
```

Comment:
it is better to store actual drbd version in values. Describe a default value for argument in openapi/values.yaml virtualization.drbdVersion. The sed in update.sh will be much more reliable.

Reply:
I think openapi spec is not a best place for storing default value because:

1) it will affect whole module values, which is not good for patch release
2) it's difficult to update with sed

I added additional helper instead
https://github.com/deckhouse/deckhouse/blob/017985f137220eecbcc4927e0feecc5a3984182e/modules/041-linstor/templates/_drbd_version.tpl#L1

Please take a look. If you don't agree I modify openapi/values of course

Reply:
It is much better, but now we reinvented templates.  We use default values for storing istio version, so it is the best of the worst options.

Reply:
Ok, done